Marlin/planner.h

```diff
@@ -166,6 +166,11 @@ class Planner {
      * Nominal speed of previous path line segment
      */
     static float previous_nominal_speed;
+	
+	/**
+	 * Limit where 64bit math is necessary for acceleration calculation
+	 */
+	static unsigned long cutoff_32bit;
```

Comment:
If we don't intent to use the variable outside the planner we don't need this statement here.
If we intend the use, the 'static' is questionable.